### PR TITLE
added error if not finished flag

### PR DESCRIPTION
**Pull request by iyeshaya**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://github.com/demisto/etc/issues/37931

## Description
added errorUnfinished flag

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [x] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonScripts/ReleaseNotes/1_3_59.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Scripts
+##### SetByIncidentId
+- Added *errorUnfinished* argument. If true, returns an error if not all the incidents were modified.
```
Comment (roysagi) [inline Packs/CommonScripts/ReleaseNotes/1_3_59.md:4]:
```suggestion
- Added the *errorUnfinished* argument - if set to true, the script will return an error if not all the incidents were modified.
```

File: Packs/CommonScripts/Scripts/script-SetByIncidentId.yml [hunk 2]
```diff
@@ -2,23 +2,27 @@ commonfields:
   id: SetByIncidentId
```
Comment (roysagi) [inline Packs/CommonScripts/Scripts/script-SetByIncidentId.yml:2]: can we try to split this script please.

Comment (iyeshaya) [reply Packs/CommonScripts/Scripts/script-SetByIncidentId.yml]: done

File: Packs/CommonScripts/Scripts/SetByIncidentId/SetByIncidentId.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  - "true"
+  - "false"
+  description: Returns an error if not all of the incidents where modified.
+  defaultValue: "false"
+scripttarget: 0
+runas: DBotRole
+runonce: false
+tests:
+- No test
+fromversion: 5.5.0
```
Comment (roysagi) [inline Packs/CommonScripts/Scripts/SetByIncidentId/SetByIncidentId.yml:44]:
```suggestion
fromversion: 5.0.0
```

**Review by roysagi — APPROVED** [review]
@iyeshaya - see my comment about fromversion before merging.

**Pull request merged**